test/prefer-add-event-listener.js

```diff
@@ -26,10 +26,10 @@ const expectedBeforeUnloadWithReturnMessage = [
 ruleTester.run('prefer-add-event-listener', rule, {
 	valid: [
 		'foo.addEventListener(\'click\', () => {})',
+		'foo.removeEventListener(\'click\', onClick)',
 		'foo.onclick',
 		'foo.setCallBack = () => {console.log(\'foo\')}',
 		'setCallBack = () => {console.log(\'foo\')}',
-		'foo.onclick.bar = () => {}',
```

Comment:
Why is this no longer valid?

Reply:
Oops, I think I might have deleted it by accident 😌.